q2_vsearch/plugin_setup.py

```diff
@@ -404,12 +406,21 @@
                   'abundances).'),
     },
     parameter_descriptions={
+        'method': ('Which algorithm to use.'),
+        # 'abskew': ('The abundance skew is used to distinguish in a threeway '
+        #            'alignment which sequence is the chimera and which are '
+        #            'the parents. The parent sequences must be this many '
+        #            'times more abundant than the child sequence to be '
+        #            'flagged as chimeric.'),
```

Comment:
Let's keep this comment out of this PR (but I'm happy to review its addition in a separate PR)